This app is broken I am fixing it

jackparsons93 opened the pull request:
This is a step from github issue 5439 It get error. Receiving NoMethodError (undefined method 'user_url' for #<Devise::RegistrationsController:0x0000000000d638>) upon successful user registration. User gets persisted in database
https://github.com/heartcombo/devise/issues/5439

jackparsons93 committed:
This app is broken I am fixing it 

This is a step from github issue 5439 It get error. Receiving NoMethodError (undefined method 'user_url' for #<Devise::RegistrationsController:0x0000000000d638>) upon successful user registration.
User gets persisted in database
https://github.com/heartcombo/devise/issues/5439

ssela98 commented:
the code you added is already in the file at [line 266](https://github.com/timothyjamesmarias/rails_7_devise_example/blob/main/config/initializers/devise.rb#L266)

jackparsons93 commented:
The line I added states  config.navigational_formats = ['*/*', :html, :turbo_stream] , line 266 is  # config.navigational_formats = ['*/*', :html]  , one it is commented out and the key difference is that I added turbo stream, that allows devise to work without a couple of different errors

ssela98 commented:
then why not comment it out and just add `:turbo_stream` to the array? like you did [here](https://github.com/heartcombo/devise/pull/5529/files) :)

jackparsons93 commented:
Thanks for the tip!

On Fri, Nov 18, 2022 at 3:42 PM Alessandra ***@***.***> wrote:

> then why not comment it out and just add :turbo_stream to the array? like
> you did here <https://github.com/heartcombo/devise/pull/5529/files> :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/timothyjamesmarias/rails_7_devise_example/pull/2#issuecomment-1320559245>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACF446UIU74W3XMAWP255ULWI7Z5RANCNFSM6AAAAAAQU6MTIU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
